doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+       caption=Axial and radial geometry of the LFR single assembly model
+
+!media lfr/annularfuel.png
+       style=width:30%
+       id=annularfuel
+       caption=LFR annular fuel pin geometry
+
+## Cross Section Generation using MC2-3
+
+It is worth explaining the procedure to generate cross sections using MC2-3 for this model. MC2-3 is one of the cross section generation modules in Griffin. The axial leakage effect needs to be considered using the MC2-3/TWODANT two-step procedure for the generation of both homogeneous cross sections for non-fuel axial regions and heterogeneous region-wise cross sections for the fuel axial region. At this point of time, the MC2-3/TWODANT procedure is not fully available in Griffin due to a licensing issue with TWODANT. Once the feature is available in Griffin, the description below will be updated.
+
+### First step id=xs1st
+
+The first step is to generate a 1041-group `rzmflx`, which is the flux solution file of a TWODANT [!citep](TWODANT) SN calculation. It contains zone-wise Legendre moments of the angular flux solution up to a specified order. TWODANT is called inside MC2-3 when its executable path is specified by the `c_twodantexe` input parameter in the `library` block of an MC2-3 input with `l_twodant=T`. If TWODANT is not available, PARTISN [!citep](PARTISN) can be used alternatively since it creates a `rzmflx` file. `c_twodant_group=BG` means that the TWODANT transport calculation is performed using a group structure specified in `c_group_structure`. `c_geometry_type=mixture` means that homogeneous materials cooresponding to axial regions are used to generate macroscopic cross sections (in the 1041-group structure in this example) to be used in the TWODANT transport calculation. 
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step1/LFR_InnerFuel_HOM_1041g_TWODANT.mcc3.sh start=\$library end=\$material include-end=False
+
+Homogenized compositions representing different axial regions are specified in the `material` block. In this step, temperature does not need to be very accurate.
+
+!listing lfr/heterogeneous_single_assembly_3D/cross_section/Step1/LFR_InnerFuel_HOM_1041g_TWODANT.mcc3.sh start=\$material end=/ include-end=True
+
+The `twodant` block is to generate a TWODANT input internally for calling TWODANT. `niso` and `ngroup` are filled by MC2-3, `mt` and `nzone` are the number of materials and coarse meshes, `im` and `jm` are the number of coarse meshes in the x and y directions, `it` and `jt` are the total number of fine meshes in the x and y directions, `igeom=6` means x-y geometry, `isn` is the Sn order, and `maxlcm` and `maxscm` are the maximum memory size of central small core memory and peripheral large core memory. `xmesh` and `ymesh` are x and y coordinates of coarse meshes, `xints` and `yints` are the number of fine meshes in each coarse x and y meshes, and `zones` are composition numbers in the `material` block assigned to each x and y coarse mesh.
```

Comment:
Is memory size in bytes?

Reply:
I think the size is in a unit of `word` = 2 bytes. I'll specify this.

Reply:
I added `(in a unit of word = 2 bytes)`.